phpunit.xml.dist

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<phpunit
+		bootstrap="tests/unit/bootstrap.php"
+		backupGlobals="false"
+		colors="true"
+		convertErrorsToExceptions="true"
+		convertNoticesToExceptions="true"
+		convertWarningsToExceptions="true"
+		verbose="true"
```

Comment:
seems like an extra tab?

Reply:
@faisal-alvi thank you, fixed that in 556b11e003f2df130ce62ed73809df2a3a6cb459